expand/expand_aws_spec.go

```diff
@@ -0,0 +1,85 @@
+package expand
+
+import (
+	"encoding/json"
+	awsAlpha1 "github.com/gardener/gardener-extension-provider-aws/pkg/apis/aws/v1alpha1"
+	corev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func getAwsConfig(aws []interface{}) *corev1beta1.ProviderConfig {
```

Comment:
In go, getters are not called `getXXX()`, the convention says getters are just called `XXX()`.
https://golang.org/doc/effective_go.html#Getters

Please rename all `getXXX()` functions to `XXX()`.